input/docs/handbook/logging/debugging-observables.md

```diff
@@ -1,22 +0,0 @@
-### Debugging Observables
```

Comment:
Sorry for the delay in reviewing this one. Just curious why this file was deleted?

Reply:
It's a duplicate (see: https://reactiveui.net/docs/handbook/logging/#debugging-observables)
but I could also be replaced with a link to the updated page.